### ISSUE TITLE
Unit listens to "queued" order - likho drops bomb early

### ISSUE THREAD

**Issue by Licho1** [ZeroK-RTS/Zero-K#3344]
1) take a likho (with firestate hold fire)
2) give it attack order to unit X 
3) shift queue attack move (fight)

now watch the behavior, likho will attack nearby enemy unit isntead of attacking the target X, wasting its shot.

**Comment by GoogleFrog** [comment]
This would be a result of that widget which sets bombers to fire at will when they have Fight commands. My guess is that the widget just listens for commands issued instead of checking the current active command.

**Comment by Licho1** [comment]
Yeah my impression too, it's also not a new bug a remember it existing for a loong time.
